pages/form/csc-case-note recording/[...stepId].jsx

```diff
@@ -0,0 +1,43 @@
+import { NextSeo } from 'next-seo';
+
+import BackButton from 'components/Layout/BackButton/BackButton';
+import FormWizard from 'components/FormWizard/FormWizard';
+import { addCase } from 'utils/api/cases';
+import form from 'data/forms/csc-case-notes-recording';
+import PersonView from 'components/PersonView/PersonView';
+
+const CscCaseNotesRecording = ({ query }) => {
+  const onFormSubmit = async (formData) => {
+    const ref = await addCase(formData.mosaic_id, {
+      caseFormData: JSON.stringify(formData),
```

Comment:
we need to send along the person and user information as we did for the other form

Reply:
> I think you mentioned but I forgot 😅
> We are waiting for the select values to be provided?
> We also need to check with the BE/DB what input names we want to use.

Yes...still working on this . the dropdowns are massive ...about 10 conditional components based on what they choose in the first select. Joy also told me late this afternoon that she is changing things so who knows.